docs-client/.eslintrc.js

```diff
@@ -0,0 +1,8 @@
+const defaultConfig = require('../settings/eslint/eslintrc');
```

Comment:
I think instead of javascript, it could be better to just use eslint's default functionality with `extends`

Reply:
I tried, but it seems like it fails to load the configs in the `extends` section, e.g. It fails to find `airbnb`. I guess it looks for the configs relatively from `../settings/eslint/` rather than from `.`.